src/themes/elife/styles.css

```diff
@@ -366,16 +405,3 @@
     list-style-type: none;
   }
 }
-
-/* Visually hide 'content header' affiliations at narrower viewports, maintaining vertical whitespace */
-@media screen and (--medium-viewport-max) {
```

Comment:
@discodavey was this removal intentional?

Reply:
Do not remember removing this

Reply:
> Do not remember removing this. Add it back in if it's needed.

Then I'm adding it back in